server/api/api.go

```diff
@@ -115,6 +116,17 @@ func (a *API) RegisterAdminRoutes(r *mux.Router) {
 
 func (a *API) requireCSRFToken(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		defer func() {
+			if p := recover(); p != nil {
+				a.logger.Error("Http handler panic",
+					mlog.Any("panic", p),
+					mlog.String("stack", string(debug.Stack())),
```

Comment:
This `debug.Stack()` and `p` variables in the logs, can lead to leaking private information into the logs?

For example, Imagine that for some reason the database fails for a second and generates a panic while the user is logging in, and the user/password data is part of the parameters in the stack trace. Can it be a problem?

Reply:
AFAIK, Go stack traces only contain pointers to func parameters.  As for the panic text, I can't imagine a case where someone is explicitly putting secrets in there. A database driver isn't going to do that.  Definitely something to watch out for, but I think we're OK here.